jdi-light-vuetify-tests/src/main/java/io/github/com/enums/Colors.java

```diff
@@ -85,6 +85,7 @@ public enum Colors {
     BLUE_ACCENT_2("rgba(68, 138, 255, 1)"),
     BLUE_ACCENT_3("rgba(41, 121, 255, 1)"),
```

Comment:
Looks like that color names are used by vuetify, so may be we can add them to vuetify package. Can be a separate PR, please check this.